site/content/docs/v1.8/upgrade-to-1.8.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Upgrading to Velero 1.8"
+layout: docs
+---
+
+## Prerequisites
+
+- Velero [v1.7.x][7] installed.
+
+If you're not yet running at least Velero v1.6, see the following:
+
+- [Upgrading to v1.1][1]
+- [Upgrading to v1.2][2]
+- [Upgrading to v1.3][3]
+- [Upgrading to v1.4][4]
+- [Upgrading to v1.5][5]
+- [Upgrading to v1.6][6]
+- [Upgrading to v1.7][7]
+
```

Comment:
There is some missing text here to link folks to the new compatibility matrix in the repo. Its in the /main upgrade file, but missing here. 

```
Before upgrading, check the [Velero compatibility matrix](https://github.com/vmware-tanzu/velero#velero-compatibility-matrix) to make sure your version of Kubernetes is supported by the new version of Velero.
```

Reply:
Added, thanks.